src/app/find/locationDetails/ServiceSection.jsx

```diff
@@ -15,7 +15,11 @@ const renderSchedule = (schedule) => {
 
   const dayNumberToName = weekday => DAYS[weekday - 1];
 
-  const formatHour = time => moment(time, 'HH:mm:ss').format('LT').replace(':00 ', ' ');
+  const formatHour = time => moment(time, 'HH:mm:ss')
+    .format('LT')
+    .replace(':00 ', ' ')
+    .replace(/(11:59 PM)|(12 AM)/, 'midnight');
```

Comment:
Would the format always be exactly this one? IIRC `LT` it a locale-aware format, meaning it would show different strings on different devices.
If that is in fact the case, I'd suggest doing the conversion to midnight while it's still a value, before the formatting turns it into a string.

Reply:
Oh, good point. I didn't realize it was locale-aware.